### PR TITLE
refactor(app): protocol details page design qa feedback 1

### LINKED ISSUE #10906: 6.0 Feedback: [Design QA] T2 Protocol Detail
### Overview

**This is not a full design QA for T2 Protocol Detail page. Due to time constraints, I'm only including what needs to be done for 6.0**

### Current Behavior

_No response_

### Expected Behavior

[Link to annotated frame in Figma ](https://www.figma.com/file/bVtCogdwiXj3ndpk4AWXCx/Milestone-1%3A-Read-and-Execute?node-id=14699%3A293314)

- [ ] (1) Update `Card__Section-r4iqug-0 itnElk` to reflect designs. Remove box-shadow, add border, border-radius 
- [ ] (2) Update `Card__Section-r4iqug-0 bopoGf` to reflect designs. Remove box-shadow, add border, border-radius 
- [ ] (3) Update to reflect 'h2-semibold' styling 
- [ ] (4) Remove box-shadow from button
- [ ] (5) Wrap text in the org/author section. This column is the same width as 'creation method' column above. That way 'description' left-aligns with 'last updated' above 
- [ ] (6) Push all these eyebrows to darkGreyEnabled
- [ ] (7) Missing the hover state on this tab. I know someone else is working on this, so please align! 
- [ ] (8) Are we supposed to have this button be enabled? In the designs it isn't. 
- [ ] (9) Fix the spacing on the open menu. Needs to be 4px below and right aligned with the button that opened it 
- [ ] (10) Make sure it's .5rem spacing between headers and the divider beneath it 
- [ ] (11) When you click 'read more', the font size for copy under "description" changes. [Loom video here. ](https://www.loom.com/share/1d258464c9a14871883a6feb1751df4a)

Screen Shot 2022-06-24 at 12 02 29 PM
Screen Shot 2022-06-24 at 12 02 47 PM
Screen Shot 2022-06-24 at 12 02 51 PM

### Steps To Reproduce

_No response_

### Operating system

_No response_

### Robot setup or anything else?

_No response_

### PR DESCRIPTION
**Pull request by jerader**
closes #10906

# Overview

address design feedback on Protocol Details page: 

Screen Shot 2022-06-28 at 1 56 47 PM

# Changelog

- changed `ProtocolDetails` and `ProtocolLabwareDetails`

# Review requests

- upload a protocol and click on the protocol card to get the protocol labware details. does it match designs?

# Risk assessment

low

### CODE REVIEW

File: app/src/organisms/ProtocolDetails/ProtocolLabwareDetails.tsx [hunk 1]
```diff
@@ -172,13 +173,13 @@ export const LabwareDetailOverflowMenu = (
...
         <Flex
-          width={'11rem'}
+          width="11rem"
           zIndex={10}
           borderRadius={'4px 4px 0px 0px'}
           boxShadow={'0px 1px 3px rgba(0, 0, 0, 0.2)'}
           position={POSITION_ABSOLUTE}
           backgroundColor={COLORS.white}
-          top="3rem"
+          top="2.3rem"
```
Comment (jerader) [inline app/src/organisms/ProtocolDetails/ProtocolLabwareDetails.tsx:186]: had to estimate this so the overflow menu appears 4px below the overflow menu button

File: app/src/organisms/ProtocolDetails/index.tsx [hunk 2]
```diff
@@ -59,12 +58,22 @@ const defaultTabStyle = css`
...
   padding: ${SPACING.spacing3} ${SPACING.spacing4};
   position: ${POSITION_RELATIVE};
 `
 
+const inactiveTabStyle = css`
+  color: ${COLORS.darkGreyEnabled};
+
+  &:hover {
+    color: ${COLORS.darkGreyEnabled};
+    background-color: #eeeeee;
```
Comment (jerader) [inline app/src/organisms/ProtocolDetails/index.tsx]: this background color `#eeeeee` isnt a constant, @mmencarelli, should I add it as a constant?

File: components/src/ui-style-constants/colors.ts [hunk 3]
```diff
@@ -5,6 +5,7 @@ export const lightBlue = '#f1f8ff'
 export const medGrey = '#e3e3e3'
 export const black = '#000000'
 export const transparent = 'transparent'
+export const fundamentalsBackgroundShade = '#eeeeee'
```
Comment (jerader) [inline components/src/ui-style-constants/colors.ts:8]: got confirmation from Mel to add this color

File: app/src/organisms/ProtocolDetails/index.tsx (hunk 2 above)
Comment (jerader) [reply app/src/organisms/ProtocolDetails/index.tsx]: just added it to our constants as `fundamentalsBackgroundShade`

**Comment by mmencarelli** [comment]
**Round 2 feedback on T2 Protocol Detail page**

[Link to annotated frame in figma](https://www.figma.com/file/bVtCogdwiXj3ndpk4AWXCx/Milestone-1%3A-Read-and-Execute?node-id=14699%3A293314)

- [x] (1) Let's get this menu overflow icon to be 4px from top and right hand side of container
- [x] (2) Can we get the open menu to be 4px from the icon? 
- [x] (3) Update margin-right on svg to be .25rem
- [x] (4) Can we fix the tiny SVG when viewport shrinks in width? 
- [x] (5) Can we fix the copy going weird when viewports shrinks in width? 
- [x] (6) should this be "loading..." too? 
- [x] (7) Update margin to this container from 24px margin top and bottom to 16px 
- [x] (8) Is it possible to remove the 8px margin-bottom on the last child here? idk if it's that that's causing the extra space? Not a huge deal, but if it's easy! 
- [x] (9) Noticing that these columns aren't hugely responsive? Could we have the columns be flex-width? 
- [x] (10) that the container as a whole doesn't abide by the 1rem margins on the entire page




Screen Shot 2022-06-28 at 2 02 42 PM
Screen Shot 2022-06-28 at 2 02 51 PM
Screen Shot 2022-06-28 at 2 02 56 PM
Screen Shot 2022-06-28 at 2 03 00 PM
Screen Shot 2022-06-28 at 2 03 49 PM

**Review by mmencarelli — CHANGES_REQUESTED** [review]
Minor edits! 

- [x] (1) Lost the left alignment of the second column!
- [x] (2) just make sure this text is all left aligned
- [x] (3) I told ya to take off too much margin! Add back in .5rem so we have 1rem margin total. 

Screen Shot 2022-06-29 at 1 59 35 PM
Screen Shot 2022-06-29 at 2 00 21 PM

**Review by mmencarelli — APPROVED** [review]
weee!!

File: app/src/organisms/ProtocolDetails/ProtocolLabwareDetails.tsx (hunk 1 above)
Comment (koji) [reply app/src/organisms/ProtocolDetails/ProtocolLabwareDetails.tsx:186]: Yeah, this happens to robot card, labware card, and protocol card.

File: app/src/organisms/ProtocolDetails/ProtocolLabwareDetails.tsx [hunk 4]
```diff
@@ -54,21 +52,26 @@ export const ProtocolLabwareDetails = (
...
+          marginBottom={SPACING.spacing3}
           data-testid={'ProtocolLabwareDetails_labware_name'}
+          width="66%"
         >
           {t('labware_name')}
         </StyledText>
         <StyledText
           as="label"
           fontWeight={TYPOGRAPHY.fontWeightSemiBold}
           color={COLORS.darkBlack}
```
Comment (koji) [inline app/src/organisms/ProtocolDetails/ProtocolLabwareDetails.tsx]: we can remove this.

File: app/src/organisms/ProtocolDetails/ProtocolLabwareDetails.tsx [hunk 5]
```diff
@@ -109,24 +112,34 @@ export const ProtocolLabwareDetailItem = (
...
+          <StyledText
+            as="p"
+            color={COLORS.darkBlack}
+            paddingRight={SPACING.spacing6}
+          >
             {displayName}
           </StyledText>
         </Flex>
-        <StyledText as="p" color={COLORS.darkBlack} marginLeft={'5rem'}>
+        <StyledText as="p" color={COLORS.darkBlack}>
```
Comment (koji) [inline app/src/organisms/ProtocolDetails/ProtocolLabwareDetails.tsx]:
```suggestion
        <StyledText as="p">
```

File: app/src/organisms/ProtocolDetails/ProtocolLabwareDetails.tsx [hunk 6]
```diff
@@ -109,24 +112,34 @@ export const ProtocolLabwareDetailItem = (
...
+              minWidth="0.75rem"
               marginRight={SPACING.spacing3}
             />
           ) : (
             <Flex marginLeft={SPACING.spacingM} />
           )}
-          <StyledText as="p" color={COLORS.darkBlack} width={SIZE_5}>
+          <StyledText
+            as="p"
+            color={COLORS.darkBlack}
```
Comment (koji) [inline app/src/organisms/ProtocolDetails/ProtocolLabwareDetails.tsx]:
```suggestion
          <StyledText
            as="p"
```

File: app/src/organisms/ProtocolDetails/ProtocolLabwareDetails.tsx [hunk 7]
```diff
@@ -165,20 +178,20 @@ export const LabwareDetailOverflowMenu = (
...
-          width={'11rem'}
+          width="11rem"
           zIndex={10}
           borderRadius={'4px 4px 0px 0px'}
           boxShadow={'0px 1px 3px rgba(0, 0, 0, 0.2)'}
           position={POSITION_ABSOLUTE}
           backgroundColor={COLORS.white}
-          top="3rem"
+          top="2.3rem"
           right={0}
```
Comment (koji) [inline app/src/organisms/ProtocolDetails/ProtocolLabwareDetails.tsx:187]: maybe `right="0"`

File: app/src/organisms/ProtocolDetails/index.tsx [hunk 8]
```diff
@@ -83,6 +92,12 @@ const currentTabStyle = css`
     width: 100%;
   }
 `
+
+const GRID_STYLE = css`
+  display: grid;
+  width: 100%;
+  grid-template-columns: 26.6% 26.6% 26.6% 20%;
```
Comment (koji) [inline app/src/organisms/ProtocolDetails/index.tsx]: The total is 99.8%. Is this on purpose?

**Review by koji — APPROVED** [review]
Looks good to me.
Left a couple of comments.

However, there is one concern about the date format. Maybe we should talk about this with the design team?
CC: @b-cooper @brenthagen @vabruzzo 

Details page `June 30, 2022 12:00`
Landing page `6/30/2022` 12:00:00

File: app/src/organisms/ProtocolDetails/index.tsx (hunk 8 above)
Comment (jerader) [reply app/src/organisms/ProtocolDetails/index.tsx]: whoops not on purpose, i just didn't do the math correctly.

**Comment by jerader** [comment]
> Details page `June 30, 2022 12:00` Landing page `6/30/2022` 12:00:00

@koji good call, I didn't even notice this - perhaps we can make a follow up ticket for when Mel and Emily are back.

File: app/src/organisms/ProtocolDetails/index.tsx (hunk 8 above)
Comment (koji) [reply app/src/organisms/ProtocolDetails/index.tsx]: > > Details page `June 30, 2022 12:00` Landing page `6/30/2022` 12:00:00
> 
> @koji good call, I didn't even notice this - perhaps we can make a follow up ticket for when Mel and Emily are back.

@jerader 
That would be a good idea.

**Review by vabruzzo — APPROVED** [review]
lgtm

**Comment by jerader** [comment]
made a follow up ticket re: the date display inconsistencies

https://github.com/Opentrons/opentrons/issues/10989

**Pull request merged**